systemverilog/SystemVerilogLexerRules.g4

```diff
@@ -434,7 +434,6 @@ STRING_LITERAL
 
 // A.9.2 Comments
 
-// commented out the following rule so that no token is generated for comments
```

Comment:
Please remove this comment and code commented below because git stores such information quite well.